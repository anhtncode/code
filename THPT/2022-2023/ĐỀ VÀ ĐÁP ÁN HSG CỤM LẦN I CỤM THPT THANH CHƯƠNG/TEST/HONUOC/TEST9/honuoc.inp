789 70
6896 95
6246 61
2205 66
2499 26
4295 76
8885 64
7130 84
6990 90
2285 2
5616 86
4752 72
6520 57
1872 85
2164 3
5294 6
9639 39
3125 10
5449 58
4716 60
5834 70
4069 78
7108 33
2842 87
8454 87
2049 32
2981 33
6673 100
1802 38
1519 23
2127 83
5236 5
4486 45
1657 54
6254 29
4850 48
5997 77
6576 10
3871 10
3874 72
6190 2
7078 100
2208 38
5221 55
1824 18
3406 33
2553 27
3239 60
9758 1
5599 56
1838 71
1893 23
5858 92
7910 95
3872 31
5179 72
6447 97
7465 78
3938 69
8914 4
3875 72
3758 30
4988 95
5816 92
4681 54
4268 67
6075 90
7834 56
1068 57
6960 43
5502 44
3593 30
8963 37
6513 61
8273 10
7626 52
1261 78
5542 23
7903 31
3362 97
2325 84
5245 96
5997 79
7724 92
6356 17
1059 22
1548 17
1778 27
8077 86
1319 70
3947 61
2115 14
9602 47
8751 39
6699 23
6864 46
8540 70
4850 75
2863 14
6498 78
7992 80
2571 58
4790 34
3576 90
5677 41
2189 16
8682 3
7259 11
3823 83
5353 67
5940 15
2193 16
9718 26
9376 59
4566 15
2684 11
5559 87
6834 68
7635 91
3886 81
2194 60
8167 68
1832 2
7762 38
8163 30
3053 54
6412 59
4810 90
2331 27
4143 28
3688 91
4984 19
2952 76
6919 59
8424 68
8495 72
2374 41
6325 85
1448 35
7503 64
9407 24
7296 33
3300 95
4353 29
5161 85
2450 56
4509 60
9928 9
9082 21
2638 28
4952 29
7071 76
6271 38
3417 48
7037 35
7190 32
1884 5
5787 100
6720 43
5993 96
2770 8
9564 68
2470 60
4399 30
9462 24
8833 98
4180 50
3402 45
9954 81
5756 33
6738 76
3649 27
8003 9
5209 2
2476 40
9609 32
7795 68
4018 51
4704 9
4167 65
9390 91
8554 79
4233 28
7106 38
9864 56
5861 35
6150 20
1354 73
3711 40
6745 16
9305 36
3608 52
7656 10
7086 46
9511 77
8883 47
3707 89
9587 75
2691 69
5207 24
1692 89
6228 10
2036 10
1608 1
8510 66
3550 37
1104 64
1344 41
8848 100
7540 56
8881 47
8555 7
6332 86
5221 47
3064 67
5325 10
9209 67
3143 4
5379 1
7594 16
6971 20
3810 58
5231 9
3995 63
7410 47
7361 3
3967 83
4779 58
5964 81
7012 79
3262 60
2197 45
6489 78
3576 50
4271 71
3144 33
6684 68
5157 54
7440 33
3093 53
3693 20
5563 35
4384 100
3567 48
7140 76
3722 78
6415 6
7209 84
5393 74
5054 79
6786 39
2390 37
9843 43
2249 66
5180 54
6820 20
5595 21
2129 57
1906 64
7756 100
1418 34
2381 77
1052 43
4809 92
9857 7
3547 5
8930 45
5168 55
5658 10
5486 86
7592 99
1474 47
7324 84
9548 65
6475 54
6593 47
8428 51
5226 31
5399 93
7621 73
4358 61
1765 36
1105 77
3081 24
2417 63
2473 97
5980 48
2949 59
1632 11
5276 46
9486 62
1627 92
1346 91
8290 21
1840 91
1331 2
1456 20
5213 5
3411 35
6207 52
6951 32
4847 44
6579 60
5255 87
9350 100
8807 45
2499 36
7110 16
2661 83
9285 18
2456 89
3040 100
9722 100
3071 68
8400 7
7387 22
6640 12
3983 58
3549 13
1050 36
4851 18
2071 93
1005 33
4591 38
6406 85
5482 15
4020 55
5437 90
2803 81
8554 4
9496 25
6407 54
2170 63
6991 28
8510 66
3585 54
3527 92
3272 19
5187 63
4505 19
7690 90
4833 32
3466 73
3095 41
9424 76
5613 49
2109 74
5642 18
1072 8
3955 37
9479 59
7463 92
8906 88
3873 35
3153 15
6934 59
3402 26
3318 44
6898 75
5414 92
3558 99
6171 65
8817 16
1060 79
4362 41
6462 54
9404 16
2702 79
3232 33
2784 16
4414 29
8785 65
9510 82
6894 84
1869 41
1735 24
4552 59
1183 38
1590 36
7804 88
7661 85
5127 74
1687 9
3145 14
8362 23
8184 59
5702 30
8851 21
1992 12
9543 75
8869 35
5934 25
4001 1
7047 8
8391 95
5079 58
5134 28
1245 18
1134 95
6520 74
8692 14
1511 51
3911 52
7808 84
2203 42
6929 34
6780 37
2825 58
3577 67
1189 76
4049 39
9974 88
1730 26
7910 94
7239 40
6672 6
8404 2
7656 36
5101 85
3158 82
6982 90
6514 96
6710 76
1051 91
6028 67
3779 97
7853 62
5460 8
6002 99
6417 69
4191 65
9041 3
6280 75
3531 84
5789 95
5817 86
2004 46
7729 26
5166 26
8609 1
4352 40
3189 88
5782 74
9869 95
1990 68
5333 36
3344 75
5474 92
3596 57
8793 21
5890 36
2289 9
4262 25
1977 75
4582 48
1907 40
7389 57
2823 53
1982 69
4881 28
1103 47
8431 6
7213 48
2649 14
5226 73
2301 63
4777 39
2199 3
2289 79
1460 88
6850 51
4948 100
7053 92
3824 51
6526 54
6828 63
5004 57
7516 88
5537 54
5891 45
2358 73
4952 51
1988 88
2292 91
4404 59
2057 65
5026 57
8322 75
9274 16
8307 39
8414 53
1867 98
4191 47
1127 77
3690 7
5114 4
3432 46
8373 16
8753 14
1196 41
1997 47
2442 97
1329 8
3796 76
6296 97
3887 56
8340 38
7570 2
5714 31
1016 58
3265 37
6882 52
2491 13
5167 48
3699 31
3366 62
5536 58
8734 33
3201 91
6965 81
3669 71
8683 39
7163 96
2330 54
9402 58
1198 79
6023 58
5583 20
4362 88
9440 24
6679 2
8669 34
3504 72
6079 80
9955 9
2543 42
8023 38
5786 12
5197 33
2431 36
9685 73
4955 3
2357 32
3256 85
5944 25
2796 14
4943 71
1560 44
9227 69
4365 67
8122 84
7363 57
9765 66
5552 22
3629 69
8035 8
9251 23
8248 26
5175 70
1294 26
9155 39
3816 14
6434 66
4384 57
5043 26
5451 82
8165 51
4141 91
1086 87
1311 9
8159 70
7132 71
8625 83
6596 62
4493 64
2974 32
4752 9
4343 8
7024 64
6072 40
7135 21
5697 68
2709 94
1950 54
1545 98
7895 70
4828 89
1125 97
6876 99
8730 84
4547 90
5163 35
9039 53
2960 35
2747 23
3204 38
6463 63
4965 61
5075 45
9217 98
2955 29
1872 25
5430 90
8774 33
4859 46
3116 91
9739 72
6611 2
9864 14
2458 63
3861 48
3952 3
1535 60
9854 22
1743 88
6017 11
1438 41
8313 17
7158 57
9661 100
1265 32
8721 37
3395 18
5426 78
8387 72
8423 13
3493 99
6134 58
3464 21
9936 31
5655 19
5251 56
3190 99
5025 15
7367 55
2546 61
2334 29
3015 98
1774 16
7361 46
4315 71
3647 69
8994 81
4397 35
1379 16
9814 31
6148 54
6234 73
3956 93
1023 47
2870 94
1019 85
1320 45
3677 11
2590 38
5997 67
4193 19
4662 98
9875 80
1033 76
4191 55
3534 51
4254 53
8342 75
3790 85
1648 91
8647 40
8316 48
6612 16
4178 4
4429 5
2562 72
4820 28
1265 29
4106 2
6693 58
6414 4
5939 46
1169 97
3126 46
7762 97
2389 58
6365 67
6497 33
8557 14
2200 92
4625 51
9879 2
9833 70
7992 49
8531 61
6907 72
4345 37
3313 97
9077 44
2733 81
6548 55
1533 10
4859 5
9038 32
3636 7
2162 84
1384 61
6647 68
6650 85
4348 99
2738 31
7801 76
7590 53
5972 40
2498 4
3895 13
7370 82
9866 36
6657 29
8238 10
2347 99
2984 43
1371 4
3691 27
7531 45
9344 46
3191 91
8380 6
6304 88
7205 66
5473 35
4524 49
4464 98
9618 47
5448 94
4048 39
4734 55
1322 26
4941 26
7954 7
8517 23
6252 15
4574 76
3771 39
7184 41
7038 27
6911 11
5829 74
4675 22
3815 80
3380 78
4771 98
5445 41
3102 44
6849 97
7732 43
7429 21
3794 54
4968 100
9693 90
9006 46
8999 33
4177 22
4430 7
1861 12
2928 94
2766 36
5831 76
1615 59
6218 70
1308 27
5442 44
5693 41
4482 34
5152 26
1273 53
5853 34
2115 45
3802 6
2070 81
8959 83
8692 6
6081 4
2005 33
2985 35
9540 64
3931 14
8620 93
6518 53
6039 83
8592 2
1234 21
2593 1
6628 58
