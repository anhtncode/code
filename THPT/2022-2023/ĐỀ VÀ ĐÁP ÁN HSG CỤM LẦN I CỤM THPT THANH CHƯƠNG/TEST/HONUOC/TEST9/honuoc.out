4122809
4082722
4042635
4002548
3962461
3922374
3882287
3842200
3802113
3762026
3721939
3681852
3641852
3602270
3562812
3523529
3484510
3445729
3407118
3368756
3330743
3292878
3255044
3217452
3180043
3142885
3106073
3069648
3033350
2997259
2961586
2926209
2891286
2856685
2822283
2788368
2755051
2722113
2689306
2656735
2624613
2592618
2560847
2529385
2498166
2467377
2436888
2406661
2376665
2347173
2318057
2289305
2260645
2232273
2204264
2176695
2149249
2121865
2094634
2067797
2041128
2014986
1989291
1963687
1938364
1913614
1889245
1865236
1841529
1818118
1795083
